Sources/Sentry/include/SentryFileManager.h

```diff
@@ -161,6 +162,12 @@ SENTRY_EXTERN void writeAppLaunchProfilingConfigFile(
  */
 SENTRY_EXTERN void removeAppLaunchProfilingConfigFile(void);
 
+SENTRY_EXTERN NSString *_Nullable sentryStaticBasePath(void);
+
+#    if defined(SENTRY_TEST) || defined(SENTRY_TEST_CI)
+SENTRY_EXTERN void removeBasePath(void);
```

Comment:
`m`: This name is a bit confusing. Maybe something like this is cleaner.
```suggestion
SENTRY_EXTERN void removeSentryStaticBasePath(void);
```